src/style.css

```diff
@@ -0,0 +1,87 @@
+body {
+  font-family: Arial, Helvetica, sans-serif;
+  font-size: 100px;
```

Comment:
assign this value to blocks where you need it

Reply:
not fixed